testbed/coav-sim.sh

```diff
@@ -21,6 +21,11 @@ SCRIPT_DIR=$(dirname $(realpath ${BASH_SOURCE[0]}))
 # Supported Autopilots: AP_PX4 and AP_APM
 AUTOPILOT=${AUTOPILOT:-"AP_PX4"}
 
+# 0: Log only
```

Comment:
I would add another level to do not log at all and redirect everything to  '/dev/null'. Is very useful when you are fixing some stuff on the script itself but dont mind about the log (like to check if something was properly killed by the script).